apps/central-scan/frontend/src/app_root.tsx

```diff
@@ -569,7 +448,7 @@ export function AppRoot({
               </React.Fragment>
             }
             unconfigureMachine={() =>
-              unconfigureServer({ ignoreBackupRequirement: true })
+              Promise.resolve(systemAdministratorUnconfigure())
```

Comment:
I manually confirmed that wrapping this method in a promise still means that errors get caught in the error boundary.